installer/gentoo/bootstrap

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh -e
+# Copyright (c) 2017 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This is a distro-specific bootstrap script, sourced from main.sh, and as such
+# has access to all of the variables set by main.sh, namely $tmp (the temporary
+# directory), $INSTALLERDIR/$DISTRO, $RELEASE, $ARCH, and $MIRROR.
+
+# Grab the latest stage3
+echo 'Finding the latest stage3...' 1>&2
+LATEST_STAGE3="$MIRROR/releases/$ARCH/autobuilds/latest-stage3-$ARCH.txt"
```

Comment:
This isn't going to work when it comes to arm or x86, since the path will look more like `$MIRROR/releases/$ARCH_FAMILY/autobuilds/latest-stage3-$ARCH.txt`